src/PickerDropdown/PickerSuggestions.js

```diff
@@ -40,6 +40,7 @@ function PickerSuggestions(
                             const menuItemProps = getItemProps({
                                 index,
                                 item,
+                                role: "menuitem",
```

Comment:
Good spot here - I assumed that the material `MenuItem` component would already have this role, but... apparently not!

Reply:
*bows* :)